src/nodes/ImageView.vue

```diff
@@ -149,6 +151,10 @@ export default {
 			}
 		},
 		imageUrl() {
+			this.setImageSrc(this.imageUrl2)
```

Comment:
Why do we have a set inside a computed?
This can potentially create an infinity loop.

Reply:
What do you think of https://github.com/nextcloud/text/commit/5ffccc411e8838cbc52ac8c2d025b3c8375bed68 ?

Reply:
It looks better to me.